app/services/catalog_record_id.rb

```diff
@@ -98,7 +103,7 @@ def find_not(type)
   end
 
   def find(type)
```

Comment:
I found it confusing that some find methods returned catalog record ids and others returned catalog records. This normalizes to returning catalog records.